README.md

```diff
@@ -11,28 +11,41 @@
 
 Besu is an Apache 2.0 licensed, MainNet compatible, Ethereum client written in Java.
 
+## Useful Links
+
+* [Besu Docs](http://besu.hyperledger.org/)
```

Comment:
You already have the `[User Documentation]` link in the doc, it would be better to reuse it. Rename it `[Besu User Documentation]` if you prefer, but having a link reference is interesting when it's reused, otherwise it's better to have the link directly close to the link text.

Reply:
You can also have different names for the same ref-style link:
```
[Besu Docs]
[Besu User Documentation][Besu Docs]
...
[Besu Docs]: http://besu.hyperledger.org/
```